Revamp app icons and about screen

m1awicki opened the pull request:
Icons styling unified.
About screen style adjusted to rest of the app.

m1awicki committed:
Revamp app icons and about screen

Icons styling unified.
About screen style adjusted to rest of the app.

m1awicki committed:
Improve and clean strings

m1awicki committed:
Update README

MDikkii commented on app/src/main/java/com/mobilecoronatracker/ui/about/AboutFragment.kt:
```diff
@@ -25,16 +24,15 @@ class AboutFragment : Fragment() {
             R.layout.fragment_about, container, false
         )
-        viewModel = CreditsListViewModel(requireActivity().application)
+        viewModel = AboutViewModel(requireActivity().application)
```
Please use ViewModelProvider or koin by viewModel to let ViewModel onCleared() be called

m1awicki replied:
done.

MDikkii commented on app/src/main/res/drawable/ic_share.xml:
```diff
@@ -1,12 +1,9 @@
-    android:height="512dp"
-    android:viewportWidth="512"
-    android:viewportHeight="512">
+    android:width="511dp"
```
You can keep 512dp here probably, or even change it to the final size of the icon (it may apply to the other icons as well)

m1awicki replied:
Fixed. Must have been accidental change during svg import.

MDikkii commented on app/src/main/res/layout/item_country_report.xml:
```diff
@@ -233,9 +233,10 @@
                     android:id="@+id/bookmark_button"
                     android:layout_width="30dp"
                     android:layout_height="30dp"
+                    android:layout_marginStart="95dp"
```
It seems to break center position of the icons, why is it added here?

m1awicki replied:
removed.

m1awicki committed:
Create AboutViewModel using DI

AboutViewModel was created manually and should be injected like
rest of the view models in the app.
Also added - minor ui adjustments.
